css/css-fonts-3/test_font_feature_values_parsing.html

```diff
@@ -152,8 +152,8 @@
   { rule: makeRule("--bongo", "@styleset { blah: 1; }"), invalid: true },
 
   /* ident tests */
-  { rule: _("@styleset { blah: 1; blah: 1; }"), serializationSame: true },
-  { rule: _("@styleset { blah: 1; de-blah: 1; blah: 2; }"), serializationSame: true },
+  { rule: _("@styleset { blah: 1; blah: 1; }"), serialization: _("@styleset { blah: 1; }") },
+  { rule: _("@styleset { blah: 1; de-blah: 1; blah: 2; }"), serialization: _("@styleset { blah: 2; de-blah: 1; }") },
```

Comment:
We should definitely use second value for `blah` here but I'm not sure about if it should be in first place or second.